notebook/_version.py

```diff
@@ -1,2 +1,28 @@
-version_info = (4, 1, 0, 'b1')
+"""
+store the current version info of the notebook.
+
+"""
+
+# Downstream maintainer, when running `python.setup.py jsversion`,
+# the version string is propagated to the JavaScript files,  do not forget to
+# patch the JavaScript files in `.postN` release done by distributions. 
+
+# Next beta/alpha/rc release: The version number for beta is X.Y.ZbN **without dots**. 
+
+version_info = (4, 1, '0b1')
```

Comment:
Sure. I opened #862 which implements this in setupbase. If you take the same function, you can move it to `_version.py` and everything will be consistent.